### PR TITLE
Add a story

### PR DESCRIPTION
**Pull request by pangduckwai**

### CODE REVIEW

**Comment by pangduckwai** [comment]
Please review this, @casey-collab

File: english/wake-up/vigilante/vigilante.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+That huge guy came out of no where and start attcking you.
+
+Being the more skillful fighter, you hit him twice and then kicked him in the face.
+
+But that seems to have little effect on him, he counter with a back-hand swing, and hit you on the side of your head.
+
+The next thing you remember is the strange dream of marshmallow and deserted island.
+
+[Lie down](wait/wait.md) again and wait for the effect of the drug to fade...
```
Comment (casey-collab) [inline english/wake-up/vigilante/vigilante.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pangduckwai** [comment]
Please review again @casey-collab

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**